### DISCUSSION TITLE
Ideas on training CRNN from experimental data

### DISCUSSION THREAD

**Discussion by jiweiqi** [DENG-MIT/CRNN#15] [Ideas]
It would be nice to study a case with experimental data. The chemical kinetic model can be very simple, although. It would be useful when you show something to others.

Maybe Ritaank and Matthew can help with it. I will also make up some candidates on this issue.

**Comment by jiweiqi** [comment]
- [Revealing hidden dynamics from time-series data by ODENet](http://arxiv.org/abs/2005.04849). Sec. 4. Application to the kinetics of actin aggregation.
   - [Protein aggregation kinetics, mechanism, and curve-fitting: A review of the literature] https://www.sciencedirect.com/science/article/pii/S1570963908003488?casa_token=jmT38Wc6tVYAAAAA:1rkLH-wRqEDDZ4BOl0BNnpFesu5IHvmYngH1kHwwTtOr9ynUaRxIZYOE9thfvXfiCIpKmb7JWw
   -

**Comment by suyongk** [comment]
One example with surface reaction:
https://link.springer.com/content/pdf/10.1007/s12274-020-2734-8.pdf
Main points:
(1) Only two surface reactions
(2) Oxidized surface is dissolved by water, and it is also modeled with surface reaction.
(3) H2 generation rate in time is given in the paper.
(4) Working conditions are not given, but it does not seem to be sensitive to temperature and pressure.
(5) No need to understand transport theory - d2 will be enough.

**Comment by jiweiqi** [comment]
> One example with surface reaction:
> https://link.springer.com/content/pdf/10.1007/s12274-020-2734-8.pdf
> Main points:
> (1) Only two surface reactions
> (2) Oxidized surface is dissolved by water, and it is also modeled with surface reaction.
> (3) H2 generation rate in time is given in the paper.
> (4) Working conditions are not given, but it does not seem to be sensitive to temperature and pressure.
> (5) No need to understand transport theory - d2 will be enough.

I guess the only thing that we need to think about is can we simulate the system using the provided reactions? As long as we can simulate the system and can generate synthesized data, we can then train CRNN.

**Comment by jiweiqi** [comment]
Experimental data on Case II of CRNN paper. @suyongk 
33. D. Darnoko, M. Cheryan, Kinetics of palm oil transesterification in a batch reactor. J. Am. Oil Chem. Soc. 77, 1263–1267 (2000). [pdf](https://link.springer.com/content/pdf/10.1007/s11746-000-0198-y.pdf)

**Comment by jiweiqi** [comment]
# Biology problem
* [Costello, Zak, and Hector Garcia Martin. "A machine learning approach to predict metabolic pathway dynamics from time-series multiomics data." NPJ systems biology and applications 4.1 (2018): 1-14.](http://www.nature.com/articles/s41540-018-0054-3)

This problem is not stiff.

Basics about MM kinetics https://chem.libretexts.org/Bookshelves/Biological_Chemistry/Supplemental_Modules_(Biological_Chemistry)/Enzymes/Enzymatic_Kinetics/Michaelis-Menten_Kinetics
